integration-tests/sanity/sanity_test.go

```diff
@@ -215,7 +215,7 @@ var _ = Describe("Sanity", func() {
 	Context("should create jiva volumes", func() {
 		It("should apply jiva pvc", func() {
 			// Getting the PVC artifact
-			artifact, err := artifacts.GetArtifactUnstructured(artifacts.JivaPVCArtifacts)
+			artifact, err := artifacts.GetArtifactUnstructuredFromFile(artifacts.JivaPVCArtifacts)
```

Comment:
Mark a TODO as these things should not be in artifacts yaml.
Only the openebs operator contents will reside in artifacts yaml

Reply:
added